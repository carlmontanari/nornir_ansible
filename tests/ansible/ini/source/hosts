[webservers]
foo.example.com
bar.example.com

[dbservers]
one.example.com
two.example.com my_var=from_hostfile
three.example.com ansible_port=5555 ansible_host=192.0.2.50

[frontend]
foo.example.com

[backend]
172.16.1.1
172.16.1.2

[servers:children]
webservers
dbservers

[servers:vars]
some_server=foo.southeast.example.com
halon_system_timeout=30
self_destruct_countdown=60
escape_pods=2
