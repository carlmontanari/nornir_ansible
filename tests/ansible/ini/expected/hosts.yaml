bar.example.com:
  connection_options: {}
  data: {}
  groups:
    - webservers
  hostname: bar.example.com
  password: null
  platform: null
  port: null
  username: null
foo.example.com:
  connection_options: {}
  data: {}
  groups:
    - frontend
    - webservers
  hostname: foo.example.com
  password: null
  platform: null
  port: null
  username: null
one.example.com:
  connection_options: {}
  data:
    my_var: from_one.example.com
  groups:
    - dbservers
  hostname: one.example.com
  password: null
  platform: null
  port: null
  username: null
three.example.com:
  connection_options: {}
  data: {}
  groups:
    - dbservers
  hostname: 192.0.2.50
  password: null
  platform: null
  port: 5555
  username: null
two.example.com:
  connection_options: {}
  data:
    my_var: from_hostfile
    whatever: asdasd
  groups:
    - dbservers
  hostname: null
  password: null
  platform: null
  port: null
  username: null
172.16.1.1:
  connection_options: {}
  data: {}
  groups:
    - backend
  hostname: 172.16.1.1
  password: null
  platform: null
  port: null
  username: null
172.16.1.2:
  connection_options: {}
  data: {}
  groups:
    - backend
  hostname: 172.16.1.2
  password: null
  platform: null
  port: null
  username: null
